src/test/java/com/epam/tc/hw2/ExerciseOneTest.java

```diff
@@ -0,0 +1,86 @@
+package com.epam.tc.hw2;
+
+import java.time.Duration;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class ExerciseOneTest extends AbstractTest {
+
+
+    private static String XPATH_ITEMS_IN_HEADER = "//ul[@class=\"uui-navigation nav navbar-nav m-l8\"]";
+    private static String XPATH_LEFT_MENU = "//*[@class='sidebar-menu left']";
+    private static String ID_FRAME_BUTTON = "frame-button";
+    private static String LIST_ITEMS_HEADER = "HOME\nCONTACT FORM\nSERVICE\nMETALS & COLORS";
+
+    @Test
+    public void exercise() {
+        //1. Open test site by URL_HOME_PAGE
+        driver.get(URL_HOME_PAGE);
+        //2. Assert Browser title "Home Page"
+        Assertions.assertThat(driver.getTitle())
+                .isEqualTo(HOME_PAGE);
+        //3. Perform login
+        WebElement loginDropdown = new WebDriverWait(driver, Duration.ofSeconds(5))
+                .until(ExpectedConditions.presenceOfElementLocated(By.xpath(XPATH_LOGIN_DROPDOWN)));
+        loginDropdown.click();
+        WebElement loginInput = driver.findElement(By.xpath(XPATH_LOGIN_INPUT));
+        loginInput.sendKeys(loginText);
+        WebElement passwordInput = driver.findElement(By.xpath(XPATH_PASSWORD_INPUT));
+        passwordInput.sendKeys(passwordText);
+        WebElement loginSubmit = driver.findElement(By.xpath(XPATH_LOGIN_SUBMIT));
+        loginSubmit.click();
+        WebElement logoutButton = driver.findElement(By.xpath(XPATH_LOGOUT_SUBMIT));
+        boolean logoutButtonIsDisplayed = logoutButton.isDisplayed();
+        Assertions.assertThat(logoutButtonIsDisplayed)
+                .isTrue();
+        //4. Assert Username is loggined
+        WebElement loginName = driver.findElement(By.id(ID_LOGIN_NAME));
+        Assertions.assertThat(loginName.getText())
+                .isEqualTo(LOGIN_ROMAN_IOVLEV);
+        //5. Assert that there are 4 items on the header section are displayed and they have proper texts
+        List<WebElement> itemsInHeader = driver.findElements(By.xpath(XPATH_ITEMS_IN_HEADER));
+        Assertions.assertThat(itemsInHeader.get(0).getText())
+                .isEqualTo(LIST_ITEMS_HEADER);
```

Comment:
`header section are displayed` where is an assert for this? Element could be found by `findElements` but not be displayed